thirdparty/junixsocket/zimbra-junixsocket/debian/control

```diff
@@ -0,0 +1,24 @@
+Source: zimbra-junixsocket
+Build-Depends: debhelper (>= 9), m4, dpkg-dev (>= 1.15.7), maven3
```

Comment:
build-depends is incorrect.  We should have no dep on maven3.  It should be zimbra-maven instead